src/IK/include/BipedalLocomotion/IK/SO3Task.h

```diff
@@ -0,0 +1,119 @@
+/**
+ * @file SO3Task.h
+ * @authors Giulio Romualdi
+ * @copyright 2021 Istituto Italiano di Tecnologia (IIT). This software may be modified and
+ * distributed under the terms of the GNU Lesser General Public License v2.1 or any later version.
+ */
+
+#ifndef BIPEDAL_LOCOMOTION_IK_SO3_TASK_H
+#define BIPEDAL_LOCOMOTION_IK_SO3_TASK_H
+
+#include <memory>
+
+#include <manif/manif.h>
+
+#include <BipedalLocomotion/IK/LinearTask.h>
+
+#include <LieGroupControllers/ProportionalController.h>
+
+
+namespace BipedalLocomotion
+{
+namespace IK
+{
+
+/**
+ * SO3Task is a concrete implementation of the IK::Task. Please use this element if you
+ * want to control the position and the orientation of a given frame rigidly attached to the robot.
+ * The task assumes perfect control of the robot velocity \f$\dot{\nu}\f$ that contains the base
+ * linear and angular velocity expressed in mixed representation and the joints velocity.
```

Comment:
I think there is a typo: "joints velocity" -> "joint velocities". I think this affects the documentation of the other tasks as well

Reply:
Here e7259e9b2dd9ea3b190127aa6bb37aee85839e5d ac2c75c243fbd825e44f6128bd2f18b3520c837a b062a812f4053688913d2ef34a96812c2d03a902 e83a331f3f2b6c6388046ba76522c8e542d436ac 73412f709b8bf6a9aa1a89bf7e07a10fcf6b3d3f 9b2e205084e86398b5a3f66c311a7dd10cdc285f